lib/rake/javaextensiontask.rb

```diff
@@ -303,5 +303,9 @@ def java_lint_arg
 
       "-Xlint:#{@lint_option}"
     end
+
+    def release_flag_supported?
+      !(RUBY_PLATFORM =~ /java/) || Gem::Version.new(Java::java.lang.System.getProperty('java.version')) >= Gem::Version.new("9")
```

Comment:
Could you use early return for readability?

```suggestion
      return false unless /java/match?(RUBY_PLATFORM)
      Gem::Version.new(Java::java.lang.System.getProperty('java.version')) >= Gem::Version.new("9")
```

Reply:
I guess I can't use ```match?``` because this gem is still ruby 1.8 compatible :) but sure, I'll change it...

Reply:
We can drop support for Ruby 1.8 because we can't test with it.

Reply:
ok, but it's for a separate PR, what should be the minimum version?

Reply:
I'm OK with a separate PR.